pkg/parallel-writer/buffer.go

```diff
@@ -0,0 +1,107 @@
+// Author: recallsong
+// Email: songruiguo@qq.com
+
+package writer
+
+import (
+	"github.com/recallsong/go-utils/errorx"
+)
+
+// Buffer .
+type Buffer struct {
+	w       Writer
+	buf     []interface{}
```

Comment:
single-letter field names are not allowed

Reply:
`w` I think is good.